eng/dependencies.props

```diff
@@ -45,12 +48,10 @@
     <CoverletConsolePackageVersion>1.4.0</CoverletConsolePackageVersion>
     <DotNetReportGeneratorGlobalToolPackageVersion>4.0.5</DotNetReportGeneratorGlobalToolPackageVersion>
 
-    <!-- Roslyn optimization data package version -->
     <MicrosoftDotNetIBCMergePackageVersion>4.6.0-alpha-00001</MicrosoftDotNetIBCMergePackageVersion>
     <TestILCAmd64retPackageVersion>$(ProjectNTfsTestILCPackageVersion)</TestILCAmd64retPackageVersion>
     <TestILCArmretPackageVersion>$(ProjectNTfsTestILCPackageVersion)</TestILCArmretPackageVersion>
     <TestILCX86retPackageVersion>$(ProjectNTfsTestILCPackageVersion)</TestILCX86retPackageVersion>
-    <OptimizationDataVersion>2.0.0-rc-61101-17</OptimizationDataVersion>
```

Comment:
Are we completely dropping the Roslyn IBC packages? Currently we don't collect data for the assemblies that Roslyn collects for, but if we are dropping their packages, we'll want to start collecting for them.

Reply:
Correct.  @tmat said that Roslyn is no longer providing new packages with training data from https://github.com/dotnet/arcade/issues/1397#issuecomment-458676749

> There is no need for a contract between Roslyn and CoreFX. It used to be that we published optimization data for SCI/SRM and other CoreFX binaries VS depends on and expected CoreFX to pick them up. That's not the case anymore.

Reply:
Thanks. I'm on an email thread where we are discussing this, so I'll let them know.

I already have a PR out to add these two assemblies back to the mix.